src/main/java/org/reactome/release/qa/common/SkipList.java

```diff
@@ -0,0 +1,78 @@
+package org.reactome.release.qa.common;
+
+import org.gk.model.GKInstance;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkipList {
+
+    public List<String> skipList = new ArrayList<>();
+
+    /**
+     * The SkipList class contains functions for reading in an assumed skiplist, and checking its contents.
+     * The main assumption is that the skiplist file will be stored in a "resources" folder, which needs to be located
+     * in the same directory as the executor, and be in the format "step_name_skip_list.txt".
+     * @param displayName - String, displayName of class calling SkipList class.
+     * @throws IOException, thrown when file does not exist. Note: Does not terminate program.
```

Comment:
The note about not terminating the program is probably better to be with the code that creates a skip list instance since the SkipList constructor can’t guarantee what the caller will do with an exception after it’s been given